### PR TITLE
Doc(eos_designs): Fix markdown errors, typos and update documentation

### PR DESCRIPTION
**Pull request by tgodaA**
## Change Summary

This PR focuses on improving eos_designs documentation and fixing most of the Markdown errors, typos in the AVD docs. Some tables in AVD docs were recreated in md format by replacing them with the old html format.

Updated first-project example with the latest knobs.

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
Improve documentation

## How to test
Not required

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/docs/contribution/setup-environment.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ $ sh -c "$(curl -fsSL https://get.avd.sh/dev/)"
 ```
 
 !!! warning
-    As devel branch is always moving on, datamodel might change and demo content could be broken. It should be used for development only.
+    As devel branch is always moving on, data-model might change and demo content could be broken. It should be used for development only.
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/contribution/setup-environment.md]: Could be rephrased to: "As the devel branch always includes the latest features and updates, the data models might change without notice and demo content could be broken. It should be used for development only."

File: ansible_collections/arista/avd/docs/contribution/setup-environment.md [hunk 2]
```diff
@@ -257,7 +257,7 @@ In a shell, install `virtualenv` package:
 $ sudo pip3 install virtualenv
 ```
 
-Create a dedicated virtual-environment where AVD will install all required Python pakages:
+Create a dedicated virtual-environment where AVD will install all required Python packages:
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/contribution/setup-environment.md]: "virtual-environment" should be "virtual environment"

File: ansible_collections/arista/avd/docs/contribution/style-guide.md [hunk 3]
```diff
@@ -38,10 +38,10 @@ When variables are used in combination with a filter, `|` shall be enclosed by s
...
 Nested jinja code block shall follow next rules:
 
-* All J2 statements must be enclosed by 1 space
-* All J2 statements must be indented by 4 more spaces within jinja delimiter
-* To close a control, end tag must have same indentation level
-* Indentation are 4 spaces and NOT tabulation
+- All J2 statements must be enclosed by 1 space
+- All J2 statements must be indented by 4 more spaces within jinja delimiter
+- To close a control, end tag must have same indentation level
+- Indentation are 4 spaces and NOT tabulation
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/contribution/style-guide.md]: "Indentation are" should be "Indentation is".

File: ansible_collections/arista/avd/docs/how-to/first-cvp-avd-project.md [hunk 4]
```diff
@@ -274,7 +275,7 @@ Power up your devices what ever the solution is. You will see them in the __`und
 
 > CloudVision might have some configlets, but none with AVD related content.
 
-### Upload custom configlets to Cloudvision.
+### Upload custom configlets to Cloudvision
 
 These configlets are managed outside of AVD project and can provide additional elements not generated by Arista Validated Design. In this example, we upload a configlet with a list of useful aliases.
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/how-to/first-cvp-avd-project.md]: should add "the" before "AVD project".

File: ansible_collections/arista/avd/docs/how-to/first-cvp-avd-project.md [hunk 5]
```diff
@@ -418,9 +419,9 @@ Go to _Provisioning > Change Control_ to create a new change control
 
 This change control is an example and you are free to build structure you want. In this scenario, all tasks can be run in parallel as we just rollout an EVPN/VXLAN fabric.
 
-## Analyze result
+### Analyze result
 
-Once devices rebooted, you can review fabric status on devices themselfs or on on CloudVision as well.
+Once devices rebooted, you can review fabric status on devices themselves or on on CloudVision as well.
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/how-to/first-cvp-avd-project.md]: Could be rephrased to "Once the devices have rebooted, you can verify the fabric status on devices themselves or on CloudVision."

File: ansible_collections/arista/avd/docs/contribution/setup-environment.md (hunk 1 above)
Comment (tgodaA) [reply ansible_collections/arista/avd/docs/contribution/setup-environment.md]: Sure, that's a good rephrase

File: ansible_collections/arista/avd/docs/contribution/style-guide.md (hunk 3 above)
Comment (tgodaA) [reply ansible_collections/arista/avd/docs/contribution/style-guide.md]: I will rephrase it to ``` Indentation should be 4 spaces and NOT tabulation```, because it is a requirement now

File: ansible_collections/arista/avd/docs/how-to/first-project.md [hunk 6]
```diff
@@ -170,78 +171,74 @@ Fabric topology is configured under `inventories/eapi-example/group_vars/AVD_FAB
...
-  bgp_as: 65001
-  nodes:
-    AVD-SPINE1:
-      id: 1
-      mgmt_ip: 10.73.254.1/24
-    AVD-SPINE2:
-      id: 2
-      mgmt_ip: 10.73.254.2/24
+  defaults:
+    platform: VEOS-LAB
```
Comment (tgodaA) [inline ansible_collections/arista/avd/docs/how-to/first-project.md:185]: I saw this to be inconsistent with respect to platform_settings. So I modified it here to make sure everyone uses the appropriate platform in their examples.

File: ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md [hunk 7]
```diff
@@ -16,77 +12,26 @@ Build configuration to publish configlets on Cloudvision.
...
-<td></td>
-<td></td>
-<td>
-    <div>File where to save information.</div>
-</td>
-</tr>
-
-</table>
-</br>
+| Parameter | Type | Required | Default | Choices | Comments |
```
Comment (tgodaA) [inline ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md]: Modified the html code to MD format.

File: ansible_collections/arista/avd/docs/modules/inventory_to_container.rst.md [hunk 8]
```diff
@@ -18,117 +14,46 @@ structure.
...
-</td>
-</tr>
-
-</table>
-</br>
-
-## Examples:
-
-    - name: generate intented variables
+| Parameter | Type | Required | Default | Choices | Comments |
```
Comment (tgodaA) [inline ansible_collections/arista/avd/docs/modules/inventory_to_container.rst.md]: Modified the html format to MD format.

File: ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md [hunk 9]
```diff
@@ -144,10 +146,6 @@ port_profiles:
         # Custom structured config added under ethernet_interfaces.<interface> for eos_cli_config_gen
         structured_config: < dictionary >
 
-      < port_profile_2 >:
```
Comment (tgodaA) [inline ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md:147]: This is the main typo that I found. A dictionary in the middle of a list. Looks like a misplaced port_profile in adapters.

File: ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md [hunk 10]
```diff
@@ -3,7 +3,7 @@
 The fabric topology variables define the connectivity between the various node types, as well as override the default switch properties.
 
 <div style="text-align:center">
-  <img src="../../../../media/5-stage-topology.gif" />
+  <img src="../../../media/5-stage-topology.gif" />
```
Comment (tgodaA) [inline ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md:6]: Missing image in the avd webpage. This should solve the problem.

File: ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md [hunk 11]
```diff
@@ -119,16 +119,16 @@ evpn_short_esi_prefix: < string, default -> "0000:0000:" >
 # l2leafs       : <subnet> + 3 + <l2leaf id>
 # GW on l2leafs : <subnet> + 1
 # Assign range larger than total l2leafs + 5
-l2leaf_inband_management_subnet: < IPv4_network/Mask >
+inband_management_subnet: < IPv4_network/Mask >
```
Comment (tgodaA) [inline ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md:122]: Replace the old knobs in the documentation with the latest ones.

File: ansible_collections/arista/avd/roles/eos_designs/doc/l3-edge.md [hunk 12]
```diff
@@ -48,9 +48,15 @@ l3_edge:
       # Enable BFD (only considered for BGP) | Optional
       bfd: < true | false | default -> false >
 
+      # Enable PTP | Optional
+      ptp_enable: < true | false | default -> false >
```
Comment (tgodaA) [inline ansible_collections/arista/avd/roles/eos_designs/doc/l3-edge.md:52]: Missing knobs.

File: ansible_collections/arista/avd/docs/contribution/overview.md [hunk 13]
```diff
@@ -141,7 +139,7 @@ The issue will be reviewed by a moderator after submission and the appropriate l
     * `Start`: Begin doing something; e.g. create a feature flag.
     * `Stop`: End doing something; e.g. remove a feature flag.
     * `Bump`: Increase the version of something e.g. dependency.
-    * `Test`: Add or refector anything regard test, e.g add a new testCases.
+    * `Test`: Add or refactor anything regard test, e.g add a new testCases.
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/contribution/overview.md]: "regard" should be "regarding"

File: ansible_collections/arista/avd/docs/contribution/overview.md [hunk 14]
```diff
@@ -151,7 +149,7 @@ The issue will be reviewed by a moderator after submission and the appropriate l
 
   * __Scopes__:
 
-    * `{{ role name }}`: AVD role impacting by PR. __Required__ for `Feat`, `Cut` and `Fix` types
+    * `{{ role_name }}`: AVD role impacting by PR. __Required__ for `Feat`, `Cut` and `Fix` types
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/contribution/overview.md]: "impacting" should be "impacted"

File: ansible_collections/arista/avd/docs/contribution/setup-environment.md [hunk 15]
```diff
@@ -73,13 +73,13 @@ $ sh -c "$(curl -fsSL https://get.avd.sh/dev/)"
...
 !!! warning
-    As devel branch is always moving on, datamodel might change and demo content could be broken. It should be used for development only.
+    As the devel branch always includes the latest features and updates, the data models might change without notice and demo content could be broken. It should be used for development only.
 
 ## Use docker as AVD shell
 
 In this approach Docker container will be leveraged to provides all the AVD requirements and playbooks and collection will be shared from your localhost to the container.
 
-This approach make the run process easier as all libraries are pre-configured in container and you can continue to use your preferred text editor to edit and build your content.
+This approach will make the run process easier as all libraries are pre-configured in container and you can continue to use your preferred text editor to edit and build your content.
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/contribution/setup-environment.md]: Add a "the" before "container" so "...are pre-configured in the container...".

File: ansible_collections/arista/avd/docs/contribution/setup-environment.md [hunk 16]
```diff
@@ -147,7 +147,7 @@ Collecting ansible==2.10
 
 ### Run AVD shell
 
-We are going to start a [new container](https://hub.docker.com/repository/docker/avdteam/base) running ansible with all the python requirements and mount local folder under `/projects`. if image is missing, docker will pull out image for you automatically.
+We are going to start a [new container](https://hub.docker.com/repository/docker/avdteam/base) running ansible with all the python requirements and mount local folder under `/projects`. If the image is missing, docker will pull it out automatically.
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/contribution/setup-environment.md]: There's no need for "out" after "docker will pull it".

File: ansible_collections/arista/avd/docs/contribution/setup-environment.md [hunk 17]
```diff
@@ -244,11 +244,11 @@ Your environment is now ready and you can start to build your own project levera
 
 ## Using Python 3 Virtual Environment feature
 
-This section describes how to configure python to run ansible and AVD.
+This section describes how to configure python to run the ansible AVD.
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/contribution/setup-environment.md]: I liked it better the way it was phrased before. Possibly rephrase to "...to run ansible-avd" or simply "...to run AVD".

File: ansible_collections/arista/avd/docs/how-to/first-project.md [hunk 18]
```diff
@@ -94,7 +94,7 @@ AVD:
         AVD_L2LEAFS:
 ```
 
-In this file, you can also define management address of all devices. Because in this lab we use Jumphost, we will configure this address in the next section, but at least we configured on which port we will connect to eAPI using `ansible_port` variable.
+In this file, you can also define management address of all devices. Because in this lab we use Jump-host, we will configure this address in the next section, but at least we configured on which port we will connect to eAPI using `ansible_port` variable.
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/how-to/first-project.md]: Add "a" before "Jump-host".

File: ansible_collections/arista/avd/docs/how-to/first-project.md (hunk 6 above)
Comment (emilarista) [reply ansible_collections/arista/avd/docs/how-to/first-project.md:185]: Looks good.

File: ansible_collections/arista/avd/docs/how-to/tower-integration.md [hunk 19]
```diff
@@ -208,12 +208,11 @@ collections:
     version: 2.1.0
 ```
 
-
 ### What to change to work with AVD and AWX
 
 Ansible has a default variable that point to inventory file used in playbook and named `{{ inventory_file }}`. Since AWX/Tower is using a database, this variable is not available anymore and [inventory file does not exist in such environment](https://github.com/ansible/awx/issues/5926).
 
-AVD use this variable to read inventory and to build container topology on Cloudvision. So to mitigate this behavior, a small warkaround is to add a task that download your inventory from your git repository and define `{{ inventory_file }}`:
+AVD use this variable to read inventory and to build container topology on Cloudvision. So to mitigate this behavior, a small workaround is to add a task that download your inventory from your git repository and define `{{ inventory_file }}`:
```
Comment (emilarista) [inline ansible_collections/arista/avd/docs/how-to/tower-integration.md]: Change to: AVD uses this variable to read the inventory and to build a container topology on Cloudvision. So to mitigate this behavior, a small workaround is to add a task that downloads your inventory from your git repository and define `{{ inventory_file }}`:

File: ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md (hunk 7 above)
Comment (emilarista) [reply ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md]: Looks good.

File: ansible_collections/arista/avd/docs/modules/inventory_to_container.rst.md (hunk 8 above)
Comment (emilarista) [reply ansible_collections/arista/avd/docs/modules/inventory_to_container.rst.md]: Looks good.

**Review by emilarista — CHANGES_REQUESTED** [review]

**Review by emilarista — DISMISSED** [review]
LGTM!

File: ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md [hunk 20]
```diff
@@ -4,10 +4,6 @@ Build arista.cvp.configlet configuration.
 
 Module added in version 2.9
```
Comment (carlbuchmann) [inline ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md]: @tgodaA - the modules documentation is auto-generated using the following script 
`ansible-avd/ansible_collections/arista/avd/docs/_build/ansible2rst.py` and generated from the python code.

Let's discuss with @titom73 if we want to continue using this approach.

File: ansible_collections/arista/avd/roles/eos_designs/doc/management-settings.md [hunk 21]
```diff
@@ -17,6 +17,7 @@ local_users:
     role: < Specify a role for the user >
     no_password: < true | do not configure a password for given username. sha512_password MUST not be defined for this user. >
     sha512_password: "< SHA512 ENCRYPTED password >"
+    ssh_key: < SSHKEY string | Optional >
```
Comment (tgodaA) [inline ansible_collections/arista/avd/roles/eos_designs/doc/management-settings.md]: Added by mistake (supported in eos_cli_config_gen not in eos_designs). I will remove this knob.

File: ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md (hunk 20 above)
Comment (tgodaA) [reply ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md]: Ok, @carlbuchmann let me know what changes should I make to this file.

File: ansible_collections/arista/avd/docs/contribution/setup-environment.md [hunk 22]
```diff
@@ -170,7 +170,7 @@ Agent pid 45
 ➜  /projects
 ```
 
-You can use a Make command to run exact same set of actions:
+You can use the Make command to run exactly the same set of actions:
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/docs/contribution/setup-environment.md]: `the exact`

File: ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md [hunk 23]
```diff
@@ -36,7 +36,7 @@ evpn_rd_type:
 # For 32-bit ASN/number the VNI can only be a 16-bit number.
 # For 16-bit ASN/number the VNI can be a 32-bit number.
 evpn_rt_type:
-  admin_subfield: < "bgp_as" | "vni" | <0-65535> | <0-4294967295> | default -> "vni" >
+  admin_subfield: < "bgp_as" | <0-65535> | <0-4294967295> | default -> <vni_id> >
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md]: just `<vni>`

File: ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md (hunk 20 above)
Comment (tgodaA) [reply ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md]: @carlbuchmann what is the issue here?

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md]: @tgoda I suggest to exclude these two MD files from your changes and get the rest merged.

Comment (tgodaA) [reply ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md]: Sure

Comment (tgodaA) [reply ansible_collections/arista/avd/docs/modules/configlet_build_config.rst.md]: Done

**Review by ClausHolbechArista — APPROVED** [review]
LGTM. Thank you for all these fixes.

File: ansible_collections/arista/avd/docs/how-to/first-cvp-avd-project.md [hunk 24]
```diff
@@ -11,33 +12,33 @@ __Demo repository:__ [arista-netdevops-community/ansible-avd-cloudvision-demo](h
 
 - A cloudVision server running version `>= 2018.2.5`
```
Comment (xaviramon) [inline ansible_collections/arista/avd/docs/how-to/first-cvp-avd-project.md]: I suggest to change to:
```
A CloudVision Portal instance running a minimum version 2018.2.5 or CloudVision as a Service (CVaaS)
```

File: ansible_collections/arista/avd/docs/how-to/first-project.md [hunk 25]
```diff
@@ -170,78 +171,74 @@ Fabric topology is configured under `inventories/eapi-example/group_vars/AVD_FAB
...
 l3leaf:
   defaults:
+    platform: VEOS-LAB
     virtual_router_mac_address: 00:1c:73:00:dc:01
-    platform: vEOS-LAB
-    bgp_as: 65100
-    spines: [AVD-SPINE1, AVD-SPINE2]
-    uplink_to_spine_interfaces: [Ethernet1, Ethernet2]
+    # VTEP VXLAN Tunnel source loopback IP for leaf switches, assigned in /32s
+    # Assign range larger then total leaf switches
```
Comment (xaviramon) [inline ansible_collections/arista/avd/docs/how-to/first-project.md]: Change 'then' for 'than'

File: ansible_collections/arista/avd/docs/how-to/first-project.md [hunk 26]
```diff
@@ -170,78 +171,74 @@ Fabric topology is configured under `inventories/eapi-example/group_vars/AVD_FAB
...
+    vtep_loopback_ipv4_pool: 192.168.254.0/24
+    loopback_ipv4_pool: 192.168.255.0/24
+    # Offset must be >= number of spines
+    loopback_ipv4_offset: 2
+    # Point to Point Network Summary range, assigned as /31 for each
+    # uplink interfaces
+    # Assign range larger then total [spines * total potential leafs * 2]
```
Comment (xaviramon) [inline ansible_collections/arista/avd/docs/how-to/first-project.md]: Change 'then' for 'than'

File: ansible_collections/arista/avd/docs/how-to/first-project.md [hunk 27]
```diff
@@ -253,15 +250,15 @@ In each variable file related to a type of devices, we have to instruct AVD what
...
 ---
-type: spine     # Must be either spine|l3leaf|l2leaf
+type: spine     # Must be either spine | l3leaf | l2leaf
 ```
 
-### Configure VNI/VLAN across the Fabric.
+### Configure VNI/VLAN across the Fabric
 
-AVD supports mechanism to create VLANs and VNIs and enable traffic forwarding in your overlay. In current version (`v1.0.2`), only following design listed below are supported:
+AVD supports mechanism to create VLANs and VNIs and enable traffic forwarding in your overlay. In current version (`v3.1.0`), only following design listed below are supported:
```
Comment (xaviramon) [inline ansible_collections/arista/avd/docs/how-to/first-project.md]: We are already in newer versions. I suggest we delete the reference to a specific version in documentation.

File: ansible_collections/arista/avd/docs/how-to/first-project.md [hunk 28]
```diff
@@ -427,12 +424,12 @@ If server is connected to only one leaf to the fabric, following template can be
...
 servers:
-  A-PR01-DMZ-POD01:     # Server name
-    rack: POD01         # Informational RACK
+  A-PR01-DMZ-POD01:                 # Server name
+    rack: POD01                     # Informational RACK
     adapters:
       - type: nic
-        server_ports: [Eth0]        # Server port to connect
-        switch_ports: [Ethernet3]   # Switch port to connect server
+        endpoint_ports: [Eth0]      # Port on server/endpoint
```
Comment (xaviramon) [inline ansible_collections/arista/avd/docs/how-to/first-project.md:434]: In the hole block, we should converge to use either server, endpoint or both to avoid any confusion.

File: ansible_collections/arista/avd/docs/how-to/first-project.md [hunk 29]
```diff
@@ -452,7 +449,7 @@ servers:
   DCI_RTR01:
     rack: DCI
     adapters:
-      - server_ports: [Eth1, Eth2]
+      - endpoint_ports: [Eth1, Eth2]
         switch_ports: [Ethernet5, Ethernet5 ]
```
Comment (xaviramon) [inline ansible_collections/arista/avd/docs/how-to/first-project.md]: Delete las space between Ethernet5 and ] to make it consistent with the whole document.

File: ansible_collections/arista/avd/docs/how-to/first-project.md [hunk 30]
```diff
@@ -520,18 +517,14 @@ Once your configuration files have been generated, you can use [`arista.avd.eos_
...
 
-The overall playbook is given for inforamtion below and you can update it to create your own workflow
+The overall playbook is given for information below and you can update it to create your own workflow
 
 ```yaml
 ---
 - name: Build Switch configuration
   hosts: all
   tasks:
-    - name: build local folders
```
Comment (xaviramon) [inline ansible_collections/arista/avd/docs/how-to/first-project.md:530]: Why is this task removed?
The playbook should include it in case the user manually deletes the output folder.

File: ansible_collections/arista/avd/docs/how-to/first-project.md (hunk 27 above)
Comment (tgodaA) [reply ansible_collections/arista/avd/docs/how-to/first-project.md]: ok

File: ansible_collections/arista/avd/docs/how-to/first-project.md (hunk 28 above)
Comment (tgodaA) [reply ansible_collections/arista/avd/docs/how-to/first-project.md:434]: I think we might deprecate ```server_ports```. So I used ```endpoint_ports``` in the example.

Comment (tgodaA) [reply ansible_collections/arista/avd/docs/how-to/first-project.md:434]: I will update the comments.

File: ansible_collections/arista/avd/docs/how-to/first-project.md (hunk 30 above)
Comment (tgodaA) [reply ansible_collections/arista/avd/docs/how-to/first-project.md:530]: This task is included in the ```eos_designs```, ```eos_cli_config_gen```, and ```eos_validate_state```. So this is no longer required.

**Review by carlbuchmann — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/docs/contribution/style-guide.md [hunk 31]
```diff
@@ -38,10 +38,10 @@ When variables are used in combination with a filter, `|` shall be enclosed by s
...
 Nested jinja code block shall follow next rules:
 
-* All J2 statements must be enclosed by 1 space
-* All J2 statements must be indented by 4 more spaces within jinja delimiter
-* To close a control, end tag must have same indentation level
-* Indentation are 4 spaces and NOT tabulation
+- All J2 statements must be enclosed by 1 space
+- All J2 statements must be indented by 4 more spaces within jinja delimiter
+- To close a control, end tag must have same indentation level
+- Indentation should be 4 spaces and NOT tabulation
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/docs/contribution/style-guide.md]: must be, not should

File: ansible_collections/arista/avd/docs/how-to/first-project.md [hunk 32]
```diff
@@ -35,7 +35,7 @@ $ tree -L 3 -d
 
 Here is a high-level overview of the topology
 
-![](../_media/avd-eos-lab-topology-overview.gif)
+![AVD eos lab topology overview](../_media/avd-eos-lab-topology-overview.gif)
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/docs/how-to/first-project.md]: EOS in caps

File: ansible_collections/arista/avd/docs/how-to/first-project.md [hunk 33]
```diff
@@ -94,27 +94,27 @@ AVD:
         AVD_L2LEAFS:
 ```
 
-In this file, you can also define management address of all devices. Because in this lab we use Jumphost, we will configure this address in the next section, but at least we configured on which port we will connect to eAPI using `ansible_port` variable.
+The management address of all devices can be defined in this file. Because a Jump-host will be used in this lab, we will configure the `ansible_port` variable to set the port number for eAPI connection, and the management address configuration is addressed in next section.
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/docs/how-to/first-project.md]: super confusing that the example above is with `ansible_host` and this section mentioned `ansible_port`. And it is not covered in any later section

File: ansible_collections/arista/avd/docs/how-to/first-project.md [hunk 34]
```diff
@@ -94,27 +94,27 @@ AVD:
...
 
-Output below is an example of how to configure EOS eAPI authentication
+An example of ansible connection parameters to establish an eAPI connection:
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/docs/how-to/first-project.md]: Ansible (capitalize)

File: ansible_collections/arista/avd/docs/how-to/tower-integration.md [hunk 35]
```diff
@@ -99,7 +99,7 @@ $ curl -X PATCH 'http://admin:password@<IP-of-AWX-INSTANCE>/api/v2/settings/syst
 }
 ```
 
-### Provision virtual-environment
+### Provision virtual environment
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/docs/how-to/tower-integration.md]: The hyphen is more correct since we are talking about the term venv / virtual-env. Not just any virtual environent

File: ansible_collections/arista/avd/docs/contribution/style-guide.md (hunk 31 above)
Comment (tgodaA) [reply ansible_collections/arista/avd/docs/contribution/style-guide.md]: Fixed. Also, I fixed the grammar(used grammarly) in this page.

File: ansible_collections/arista/avd/docs/how-to/first-project.md (hunk 32 above)
Comment (tgodaA) [reply ansible_collections/arista/avd/docs/how-to/first-project.md]: Done

File: ansible_collections/arista/avd/docs/how-to/first-project.md (hunk 33 above)
Comment (tgodaA) [reply ansible_collections/arista/avd/docs/how-to/first-project.md]: I didn't pay attention. I just para-phrased it. Will fix it.
